core/src/test/java/dev/morphia/mapping/codec/MorphiaCodecProviderTest.java

```diff
@@ -0,0 +1,29 @@
+package dev.morphia.mapping.codec;
+
+import java.util.List;
+
+import dev.morphia.Datastore;
+import dev.morphia.test.TestBase;
+
+import org.bson.codecs.pojo.PropertyCodecProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class MorphiaCodecProviderTest extends TestBase {
+
+    @Test
+    public void NewMorphiaCodecProvider_NoOpPropertyCodecProviderInConfig_ExpectedOrderOfProvidersReturned() {
```

Comment:
this is quite the mouthful.  maybe a simpler name?  maybe i'll just rename after the merge.

Reply:
Fine by me. Sometimes I go a little overboard with test naming.